{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/assets/scss/_base.scss

```diff
@@ -0,0 +1,10 @@
+//fixes horizontal scroll issue cause by header+footer width
+html,
+body {
+    overflow-x: hidden;
+}
+
+//this is to position the footer at bottom of page with little content, the 100px is the height of the footer
```

Comment:
I am confused about the numbers. Here it's 150px substracted and in the footer itself the height is 50px?

Reply:
sorry, this is height of header and footer combined

Reply:
Ah! Alright!